### ISSUE TITLE
Improvement: Delelte Old Messages / Timeout for logger

### ISSUE THREAD

**Issue by Snipeeeey** [Rdimo/Hazard-Token-Grabber-V2#466]
So idk if I'm the only one who is annoyed by this, but like when a person runs the .exe I keep getting their info again. Which is a good thing, BUT when a person runs the .exe like 1321321 times the webhook gets "spammed" with the new info. 
I think a timeout feature or smth would be a great addition. lmk what you think

**Comment by timmywag** [comment]
I suggest making the grabber delete the program after running.

**Comment by Smug246** [comment]
> I suggest making the grabber delete the webhook after running.
> 
> 
> 
> 

nah because u the people u send it to might send it to other people. so it spreads. shit idea timmy

**Comment by Smug246** [comment]
> So idk if I'm the only one who is annoyed by this, but like when a person runs the .exe I keep getting their info again. Which is a good thing, BUT when a person runs the .exe like 1321321 times the webhook gets "spammed" with the new info. 
> 
> I think a timeout feature or smth would be a great addition. lmk what you think

i get your point but you can't rly complain about grabbing someone lots of times. realistically someone isn't going to open it more than 5 times before you've logged in with there info and they realise there fucked. it's a good idea but not very necessary.

**Comment by timmywag** [comment]
> > I suggest making the grabber delete the webhook after running.
> 
> nah because the people u send it to might send it to other people. so it spreads. shit idea timmy

makes sense if your using a build bot, but your too slow to make one so nvm

**Comment by Smug246** [comment]
> > > I suggest making the grabber delete the webhook after running.
> 
> > 
> 
> > nah because the people u send it to might send it to other people. so it spreads. shit idea timmy
> 
> 
> 
> makes sense if your using a build bot, but your too slow to make one so nvm


99% of people can't install a module let alone use a build bot. shit idea timmy

**Comment by timmywag** [comment]
> > > > I suggest making the grabber delete the webhook after running.
> > 
> > 
> > > 
> > 
> > 
> > > nah because the people u send it to might send it to other people. so it spreads. shit idea timmy
> > 
> > 
> > makes sense if your using a build bot, but your too slow to make one so nvm
> 
> 99% of people can't install a module let alone use a build bot. shit idea timmy

your mother giving birth to you was a bad idea. shit idea smuggy

**Comment by Smug246** [comment]
> > > > > I suggest making the grabber delete the webhook after running.
> 
> > > 
> 
> > > 
> 
> > > > 
> 
> > > 
> 
> > > 
> 
> > > > nah because the people u send it to might send it to other people. so it spreads. shit idea timmy
> 
> > > 
> 
> > > 
> 
> > > makes sense if your using a build bot, but your too slow to make one so nvm
> 
> > 
> 
> > 99% of people can't install a module let alone use a build bot. shit idea timmy
> 
> 
> 
> your mother giving birth to you was a bad idea. shit idea smuggy

ye f u mum. shit idea mum

**Comment by timmywag** [comment]
> > > > > > I suggest making the grabber delete the webhook after running.
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > >
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > > nah because the people u send it to might send it to other people. so it spreads. shit idea timmy
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > makes sense if your using a build bot, but your too slow to make one so nvm
> > 
> > 
> > > 
> > 
> > 
> > > 99% of people can't install a module let alone use a build bot. shit idea timmy
> > 
> > 
> > your mother giving birth to you was a bad idea. shit idea smuggy
> 
> ye f u mum. shit idea mum

love you babe

**Comment by Snipeeeey** [comment]
> 

I get your point aswell, but like some people really run the grabber like 40 times. So imo a timeout feature would be great

**Comment by timmywag** [comment]
nah stfu ong ong frfr

**Comment by Smug246** [comment]
> > 
> 
> 
> 
> I get your point aswell, but like some people really run the grabber like 40 times. So imo a timeout feature would be great

there dumb mfs but ye ig. not sure how you could implement that

**Comment by timmywag** [comment]
shut up smuggy

**Comment by Snipeeeey** [comment]
timmy you have no swag

**Comment by timmywag** [comment]
shut up skid 
your literally mad someone caused a minor inconvenience
close this shit 

> timmy you have no swag

**Comment by Snipeeeey** [comment]
L

**Comment by GodFather227** [comment]
Technically u can just open the core folder and go into the index.js and check if the injection is there @timmywag soooo smh, Smd, your fat

**Comment by timmywag** [comment]
> Technically u can just open the core folder and go into the index.js and check if the injection is there @timmywag soooo smh, Smd, your fat

that isn't what this issue is about nigga
read 💀

**Comment by GodFather227** [comment]
> > Technically u can just open the core folder and go into the index.js and check if the injection is there @timmywag soooo smh, Smd, your fat
> 
> that isn't what this issue is about nigga read 💀

Wait what

**Comment by GodFather227** [comment]
Ooooh like it’s added to startup

**Comment by GodFather227** [comment]
U can do the same with that ig

**Comment by timmywag** [comment]
smh

**Comment by Rdimo** [comment]
> So idk if I'm the only one who is annoyed by this, but like when a person runs the .exe I keep getting their info again. Which is a good thing, BUT when a person runs the .exe like 1321321 times the webhook gets "spammed" with the new info. I think a timeout feature or smth would be a great addition. lmk what you think

ye rly good idea, the first versions of v2 did this, only allowed 1 run per pc. But then disabled it for more flexibility. Very easily added tho and gonna make it optional ofc, good idea

**Comment by PandoricGalaxy** [comment]
Umm correct me if I'm dumb but can't a discord bot be made to delete the old message of the grab in the server by looking at the ip or name of pc or hwid. This way u can get new grabs if there are m
New passwords or anything in it's pc. Otherwise if u want that only 1 time a grna comes from the person I'll try my best (only the .exe to stop spamming again and again not the injection cause I'm not good)

**Comment by timmywag** [comment]
too much effort for such a minor issue fr

**Comment by PandoricGalaxy** [comment]
> too much effort for such a minor issue fr

I edited it + it's not minor issue, it's a good suggestion

**Comment by timmywag** [comment]
its minor nigga
your trippin if you think its a good idea to make a full bot for spam reduction
infact it doesnt even fix that issue it just hides the spam

plus, it would just make the helpers jobs harder

**Comment by PandoricGalaxy** [comment]
> its minor nigga your trippin if you think its a good idea to make a full bot for spam reduction infact it doesnt even fix that issue it just hides the spam
> 
> plus, it would just make the helpers jobs harder

Well read the second part of what I wrote.. implement check to see if the user is already grabbed then skip grabber code and run other code if there.... Ik how I'll do that but only if the issue owner wants

**Comment by timmywag** [comment]
sounds like a you problem

**Comment by PandoricGalaxy** [comment]
> sounds like a you problem

Dude i didn't even ask u anything about it...u just poked ur nose and said it's my problem

**Comment by timmywag** [comment]
idk man, seems like a you problem

**Comment by PandoricGalaxy** [comment]
> idk man, seems like a you problem

@Rdimo make me mod for these issues imma do some stuff like closing empty ones
